spec/controllers/trainer_controller_spec.rb

```diff
@@ -65,6 +65,17 @@
         card = create_and_check_review_card(@user, @block, 5, 'house')
         expect(card.review_step).to eq(5)
       end
+
+      it 'set review_date step=1 levenshtein_distance=1' do
+        card = create_and_check_review_card(@user, @block, 1, 'hous')
+        expect(card.review_date.strftime('%Y-%m-%d %H:%M')).
+            to eq((Time.zone.now + 12.hours).strftime('%Y-%m-%d %H:%M'))
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.